packages/tradershub/tailwind.config.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
                         'primary-background': '#0e0e0e',
                         'prominent-text': '#ffffff',
                         'secondary-background': '#151717',
+                        'less-prominent': '#999999',
```

Comment:
Will the dark `less-prominent` hex code will be the same as the light?

Reply:
We don't have the dark theme deign but let me change it to the same color we have in V1